prototype/core/impl/pom.xml

```diff
@@ -53,6 +53,10 @@
       <groupId>org.geckoprojects.emf</groupId>
       <artifactId>org.gecko.emf.osgi.component</artifactId>
     </dependency>
+    <dependency>
+      <groupId>biz.aQute.bnd</groupId>
+      <artifactId>aQute.libg</artifactId>
+    </dependency>
```

Comment:
I don't think we should be adding dependencies on bnd libraries unless we really need them. This looks like it's only there for `Exceptions.duck.